### PR TITLE
Nodejs week 3 

### PR DESCRIPTION
**Pull request by Hani-far**

### CODE REVIEW

**Comment by Hani-far** [comment]
@cecastosic, hi, I made this PR with the applied changes for nodejs/week3 HW . may I ask you to look at it whenever you have time? Thanks in advance.

File: nodejs/week3/warmup/index.js [hunk 1]
```diff
@@ -23,19 +27,67 @@ router.get("/add", (request, response) => {
...
+            .reduce((accumulator, currentValue) => accumulator.concat(currentValue), [])
+            .map((param) => Number(param))
+            .reduce((previous, current) => previous - current);
+        sendResponse(difference, response);
     } catch (error) {
         response.status(404).send("values undefined");
     }
 })
 
+router.get("/divide", (request, response) => {
```
Comment (orhantoy) [inline nodejs/week3/warmup/index.js]: This route should be called `/division`, just like the similar POST route.

**Review by orhantoy — APPROVED** [review]
Nice work 💯

File: nodejs/week3/warmup/index.js (hunk 1 above)
Comment (Hani-far) [reply nodejs/week3/warmup/index.js]: Thanks Orhan for your time and feedback

**Pull request merged**